bin/boot

```diff
@@ -25,10 +27,46 @@ function login  {
 }
 
 function start {
-    echo "Running odoo..."
+    echo $log_src[`date +%F.%H:%M:%S`]' Updating Odoo conf...'
+    if [ "$ADMIN_PASSWORD" ]; then
+        sed -i "s/_ADMIN_PASSWORD_/$ADMIN_PASSWORD/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$DB_USER" ]; then
+        sed -i "s/_DB_USER_/$DB_USER/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$DB_PASSWORD" ]; then
+        sed -i "s/_DB_PASSWORD_/$DB_PASSWORD/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$ADDONS_REPO" ]; then
+        echo $log_src[`date +%F.%H:%M:%S`]' Setting up SSH key'
+        cp -R /mnt/ssh /root/.ssh
+        chown -R $USER:$USER /root/.ssh
+        chmod 600 /root/.ssh/*
+        ssh-keyscan github.com  >> /root/.ssh/known_hosts
+        echo $log_src[`date +%F.%H:%M:%S`]' Downloading additional addons...'
+        python /mnt/scripts/addons.py
+    else
+        echo $log_src[`date +%F.%H:%M:%S`]' No additional addons to download'
+        python /mnt/scripts/noaddons.py
```

Comment:
And add a FIXME on top of this line to indicate that this script should be replaced with the code from commit 6076053f70052bd3a6c92a4c4c28d6c99b30d80b